app/components/common/Feedback.tsx

```diff
@@ -202,44 +200,12 @@ const IssueDescriptionSection = styled.div`
 interface FormFields {
   name: string;
   email: string;
-  description: string;
+  comments: string;
 }
 
-const sendReport = (
-  name: string,
-  email: string,
-  comments: string,
-  uniqID: string
-) =>
-  fetch(
-    `https://sentry.io/api/0/projects/${process.env.SENTRY_SLUG}/${process.env.SENTRY_PROJECT_SLUG}/user-feedback/`,
-    {
-      method: 'POST',
-      headers: {
-        Authorization: `DSN ${process.env.SENTRY_DSN}`,
-        'Content-Type': 'application/json',
-      },
-      body: JSON.stringify({
-        event_id: captureReactMessage(`
-           User has submitted an issue and asked to check it. id: ${uniqID}
-           ------------------------------------------------------------------
-           ${name}
-           ------------------------------------------------------------------
-           ${email}
-           ------------------------------------------------------------------
-           ${comments}
-           ------------------------------------------------------------------
-           `),
-        name,
-        email,
-        comments,
-      }),
-    }
-  );
-
 const DESCRIPTION_PLACEHOLDER = `### Describe the bug
 
-A clear and concise description of what the bug is.
+A clear and concise comments of what the bug is.
```

Comment:
```suggestion
A clear and concise description of what the bug is.
```
I guess find&replace oversight